From d4b3b521cc398d85340075d99a2bf6964f13b544 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Sat, 24 Apr 2021 11:38:58 +0200
Subject: [PATCH] Hide kdoctools_install() behind KF5DocTools_FOUND conditional

The dependency is optional in this package so the macro call must be too.

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2e2512f..3d6c86a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -120,5 +120,7 @@ ecm_qt_install_logging_categories(
 
 kde_configure_git_pre_commit_hook(CHECKS CLANG_FORMAT)
 ki18n_install(po)
-kdoctools_install(po)
+if(KF5DocTools_FOUND)
+    kdoctools_install(po)
+endif()
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
-- 
2.31.1

